src/applications/search/utils.js

```diff
@@ -16,3 +16,8 @@ export function truncateResponseString(string, maxLength) {
 export function removeDoubleBars(string) {
   return string.replace('| Veterans Affairs', '');
 }
+
+export function isSearchStrInvalid(str) {
+  const formattedStr = str.trim();
+  return !formattedStr.length > 0 || !formattedStr.length > 255;
```

Comment:
Seems like we could use 

```!formattedStr.length || formattedStr.length > 255```

Isn't `!formattedStr.length > 255` actually valid input? I tested in the review instance and it seems to be working but I'm not sure why.

Reply:
Thanks Randi. This is a better solution. This util function is only used for the call to the API. We still have the length string logic in place from before, so the correct error message will display for the long string length.